src/Firehose.Web/Authors/MarkDeanilVicente.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using Firehose.Web.Infrastructure;
+
+namespace Firehose.Web.Authors
+{
+    public class MarkDeanilVicente : IAmAMicrosoftMVP, IAmAXamarinMVP
+    {
+        public string FirstName => "Mark Deanil";
+        public string LastName => "Vicente";
+        public string StateOrRegion => "Manila, Philippines";
+        public string EmailAddress => "markdeanilvicente@gmail.com";
+        public string ShortBioOrTagLine => "Mark Deanil Vicente is a Microsoft MVP. He's currently working as programmer & a developer. He's also one of the founders of Mobile .Net Developers PH (MondPH) Community. He enjoys drinking wine and hot tea";
```

Comment:
Your bio is too long. It breaks the layout of the authors page. Please keep it short and concise.